scripts/rspec/rspec.ps1

```diff
@@ -224,6 +233,7 @@ if ($ClassName -And $RuleKey) {
     elseif ($Language -eq "vbnet") {
        GenerateRuleClassesVB
     }
+    CopyTestCasesFromRspec $ClassName "${RspecRepositoryPath}\\rules\\${RuleKey}\\csharp" $TestCasesFolder
```

Comment:
`CopyTestCasesFromRspec` should be called before generating the default TestCases file.
Update the TestCase file creation to not be generated if there is already a TestCase with the same name.